test/functional/desired.js

```diff
@@ -63,6 +63,7 @@ const GPS_DEMO_CAPS = amendCapabilities(GENERIC_CAPS, {
 
 const BROWSER_CAPS = amendCapabilities(GENERIC_CAPS, {
   browserName: 'Chrome',
+  'appium:noReset': true,
```

Comment:
Is this necessary?

Reply:
I think no, I don't remember how I did it... I removed it 👍

Reply:
https://github.com/appium/appium-uiautomator2-driver/pull/583/commits/663067b4ebacd127ec1c86509f7bb65aa6c1fd79